docs/adguard-vpn-for-windows/solving-problems/logs.md

```diff
@@ -39,6 +39,6 @@ In most cases, the default logging level is sufficient to trace down possible bu
 
 6. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred.
 
-7. Make sure that there is a check mark next to **Attach technical logs** and click **Send**.
+7. Make sure that there is a check mark next to **Send detailed system info** and click **Send**.
 
->If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, select **Settings** → **General** → **Export logs** in the **Actions** section on the right.
\ No newline at end of file
+>If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.
```

Comment:
I've checked - all the wording is consistent with what we use in our apps. If you want us to change the wording, write us via Crowdin.

Reply:
@anastasiia-fedotova I already reported in Crowdin as a mistake issue you can look at these and also sent a message.
As you can see its correct for Windows: 

![image](https://user-images.githubusercontent.com/35523759/194091023-afaba2dc-48f2-491d-9c3e-207d1515970f.png)